internal/cli/atlas/events/list.go

```diff
@@ -127,7 +142,19 @@ func (opts *ListOpts) NewProjectListOptions() admin.ListProjectEventsApiParams {
 		p.IncludeCount = pointer.Get(false)
 	}
 
-	return p
+	return p, nil
+}
+
+func ParseDate(date string) (*time.Time, error) {
```

Comment:
```suggestion
func parseDate(date string) (*time.Time, error) {
```
let's make it private